.github/actions/test/action.yaml

```diff
@@ -0,0 +1,15 @@
+# action.yml
+name: 'Run sbt test'
+description: 'Using a container with old code, run the tests on new code.'
```

Comment:
This is just for educating me. Would you elaborate why using old image would fulfill our need to prevent running untrusted code?

Also if we choose to change dependency version in the future, will this breaks?

Reply:
You know, thanks for asking.  The answer is that this ensures that the container build is being done from safe code, and the container build is the part with access to secrets - but the fact that the job triggered even though it isn't submitted yet suggests that's not right!  I didn't expect that, and that's disappointing.  I'll do some more research on the permissions model.

Reply:
Removed - it's a little simpler now.